content/blog/2021-01-11-december-release/contents.lr

```diff
@@ -0,0 +1,119 @@
+title: InvenioRDM Alpha 14 (December Release)
+---
+author: Guillaume Viger, Sara Gonzales
```

Comment:
Hmm I think we all contributed to it. Most content was by Lars I think (I did a lot of edits :man_shrugging: ): `Lars Holm Nielsen, Sara Gonzales`.

Reply:
Please add names of all who contributed!